### PR TITLE
[release] add sonarqube release notes

### PR DESCRIPTION
**Pull request by weeyum**
Screen Shot 2021-04-06 at 11 56 55 PM

### CODE REVIEW

File: product-releases/2021-04-08-sonarqube-support.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+author_url: https://github.com/weeyum
+author_image_url: https://assets-global.website-files.com/5fd1d1ab54430e6b747263b2/5fd1d1ab54430ee03472641b_william%20headshot.png
+tags: []
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+In the past few days, the effx platform engineering team landed support for [Sonarqube][].
+Sonarqube is an open-source platform developed by SonarSource for continuous inspection of code quality.
+Sonar does static code analysis, which provides a detailed report of bugs, code smells, vulnerabilities, code duplications.
```
Comment (mjpitz) [inline product-releases/2021-04-08-sonarqube-support.mdx:14]: `Sonar performs static code analysis`

Comment (mjpitz) [reply product-releases/2021-04-08-sonarqube-support.mdx:14]: I'd also add a line after this about how effx uses this information and turns it into more value for the customer. I was having trouble coming up with a line for this which was why I started looking at adding tags.

File: product-releases/2021-04-08-sonarqube-support.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+[Sign up][] today and give it a try!
+You can even leverage our [quick start][] guide to get up and running in no time.
+
+Already a customer?
+Head on over to your [integrations][] page to learn more about how to set it up.
+
+<p></p>
+
+[Sign up]: https://app.effx.com/sign_up
+[quick start]: /docs/integrations/prometheus#quick-start-with-minikube
```
Comment (mjpitz) [inline product-releases/2021-04-08-sonarqube-support.mdx:45]: Do you plan on replacing this?

File: product-releases/2021-04-08-sonarqube-support.mdx (hunk 1 above)
Comment (weeyum) [reply product-releases/2021-04-08-sonarqube-support.mdx:14]: same, the current approach doesn't add too much yet. perhaps we can come back and update the value prop once we have a better selling point? 

I think ideally it'd work similar to SLOs with badges

**Review by Ttsunekawa — APPROVED** [review]

File: product-releases/2021-04-08-sonarqube-support.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+In addition to detecting services, job statuses sent to effx are surfaced in your service activity feed.
+
+<p>
+  <img alt="feed screenshot"
+      src={useBaseUrl('img/integrations/sonarqube-feed.png')}/>
+</p>
+
+Curious about how it works?
+[Sign up][] today and give it a try!
+You can even leverage our [quick start][] guide to get up and running in no time.
```
Comment (mjpitz) [inline product-releases/2021-04-08-sonarqube-support.mdx:37]: missing link

**Pull request closed without merging**